schema/connector-discovery-schema.json

```diff
@@ -0,0 +1,297 @@
+    },
+    "config": {
+      "$ref": "#/definitions/configs",
+      "description": "Multi tenant connector config properties that admin will be prompted while configuring Mobile Flows."
+    }
+  },
+  "definitions": {
+    "link": {
+      "type": "object",
+      "properties": {
+        "href": {
+          "type": "string",
+          "format": "uri"
+        }
+      },
+      "required": [
+        "href"
+      ]
+    },
+    "object_types": {
+      "type": "object",
+      "additionalProperties": {
+        "$ref": "#/definitions/object_type",
+        "description": "Each additional property is an object type."
+      }
+    },
+    "object_type": {
+      "type": "object",
+      "properties": {
+        "doc": {
+          "$ref": "#/definitions/link",
+          "description": "Link to the schema of the object type."
+        },
+        "fields": {
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/input_field",
+            "description": "Each additional property describes an input field."
+          }
+        },
+        "pollable": {
+          "type": "boolean",
+          "description": "A hint for the client that polling for this object_type might produce objects containing new data."
+        },
+        "metadata": {
+          "type": "object",
+          "description": "Free form key-value pairs. Can contain additional metadata about the object_type, if anything is required by client."
+        },
+        "endpoint": {
+          "$ref": "#/definitions/link",
+          "description": "Link to an endpoint that produces this object_type."
+        }
+      },
+      "required": [
+        "endpoint"
+      ]
+    },
+    "input_field": {
+      "type": "object",
+      "properties": {
+        "regex": {
+          "type": "string",
+          "description": "Regular expression that can be used by client to capture valid inputs, from a text line or paragraph."
+        },
+        "capture_group": {
+          "type": "integer",
+          "description": "Which parentheses group of the regex captured text to be used."
+        },
+        "env": {
+          "type": "string",
+          "description": "To be collected from client's Environment variables and send in the request. See Discovery in wiki."
+        },
+        "required": {
+          "type": "boolean",
+          "description": "A flag says if this input field is mandatory. If client knows this field then it can include in the request for better results."
+        }
+      },
+      "dependencies": {
+        "regex": {
+          "required": [
+            "capture_group"
+          ]
+        }
+      }
+    },
+    "actions": {
+      "type": "object",
+      "additionalProperties": {
+        "$ref": "#/definitions/connector_level_action",
+        "description": "Each additional property is a connector level action."
```

Comment:
Description doesn't sound appropriate.

Reply:
Are you talking about the term "connector level action" or we will change something like below.
- "Each additional property describes about connector level action" ?

Reply:
Yes.